src/components/SearchInput.vue

```diff
@@ -45,7 +45,7 @@
 }
 
 .search__icon { @apply
-  text-foreground-400
-  ml-2
+  ml-2;
+  color: var(--p-color-input-placeholder);
```

Comment:
should there be a utility class for placeholder?

Reply:
So this is a component level token, in general I've kept those as css variables. If placeholder keeps popping up for some reason, we could consider making that more generic.